app/components/inputgroup/inputgroup.js

```diff
@@ -40,7 +40,12 @@ const InputGroup = React.createClass({
     rows: React.PropTypes.number,
     disabled: React.PropTypes.bool,
     multi: React.PropTypes.bool,
-    onChange: React.PropTypes.func
+    onChange: React.PropTypes.func,
+    info: React.PropTypes.string,
+  },
+
+  defaultProps: {
+    info: undefined,
```

Comment:
```suggestion
```
default properties are set when the values are undefined,
this property has no effect.